starboard/android/shared/video_render_algorithm.cc

```diff
@@ -60,13 +60,22 @@ void VideoRenderAlgorithm::Render(
       break;
     }
 
+    if (first_frame_released_ == false) {
+      // Draw the first frame immediately after it's received.
+      auto status = draw_frame_cb(frames->front(), GetSystemNanoTime());
```

Comment:
Will this cause a key frame earlier than the seek target to be rendered?

Reply:
Yes, it will and is not desired, so I removed the code.